src/elements/content-sidebar/activity-feed/task-new/AssigneeList.scss

```diff
@@ -13,20 +13,17 @@ $avatar-size: 28px;
     display: flex;
     margin: 6px 0 0 0;
     padding: 3px 0;
+}
 
-    .bcs-AssigneeList-detailsStatus {
-        color: $bdl-neutral-02;
-        font-size: 12px;
-    }
+.bcs-AssigneeList-detailsStatus {
+    color: $bdl-neutral-02;
+    font-size: 12px;
```

Comment:
It's funny that we have almost equal number of usages of 11px as 12px across similar use cases. We probably need to to scrub one of these sizes away...